src/main/kotlin/net/atos/zac/app/zaak/model/RESTZaakAfbrekenGegevens.kt

```diff
@@ -10,5 +10,5 @@ import nl.info.zac.util.NoArgConstructor
 @NoArgConstructor
 @AllOpen
 data class RESTZaakAfbrekenGegevens(
-    var zaakbeeindigRedenId: Long
+    var zaakbeeindigRedenId: Long?
```

Comment:
Nullable? Rather use a static value for the static value nor in the database, -1 for instance...

Reply:
Using `String` for id now